service/src/main/kotlin/fi/espoo/evaka/attendance/ChildAttendance.kt

```diff
@@ -12,6 +12,40 @@ import java.time.Instant
 import java.time.LocalDate
 import java.util.UUID
 
+data class ChildResult(
+    val status: ChildResultStatus,
+    val child: ChildSensitiveInformation? = null
+)
+
+enum class ChildResultStatus {
+    SUCCESS, WRONG_PIN, NOT_FOUND
+}
+
+data class ChildSensitiveInformation(
+    val id: UUID,
+    val firstName: String,
+    val lastName: String,
+    val preferredName: String?,
+    val ssn: String?,
+    val childAddress: String?,
+    val placementTypes: List<PlacementType>?,
+    val allergies: String?,
+    val diet: String?,
+    val medication: String?,
+    val contact1: ContactInfo?,
+    val contact2: ContactInfo?,
+    val backupPickup1: ContactInfo?,
+    val backupPickup2: ContactInfo?
+)
+
+data class ContactInfo(
+    val firstName: String?,
```

Comment:
Do we really need to handle NULLs and empty strings separately? IMHO it's generally simpler to use empty strings for values that haven't been set